securedrop/journalist_templates/_confirmation_modal.html

```diff
@@ -1,13 +1,15 @@
 <div id="{{ modal_data.modal_id }}" class="modal-dialog">
   <a href="#close" class="external"></a>
   <div>
-    <a href="#close" title="{{ gettext('Close') }}" class="close">X</a>
+    <a href="#close" title="{{ gettext('Close') }}" class="close"> <img src="{{ url_for('static', filename='i/modal-x-white.png') }}" height="12" width="12"></a>
```

Comment:
The changes and styling here look great:

This PR:
![delete-source-modal](https://user-images.githubusercontent.com/15223328/107976389-78a0bc00-6f87-11eb-9c44-3c2f85b6e872.png)


Prototype
![delete-source-modal-proto](https://user-images.githubusercontent.com/15223328/107976401-7dfe0680-6f87-11eb-9a33-09ce168254fc.png)

The only detail that appears off is the blue circle in which the X is located top-right, it looks like the colors don't match the `h2` from the prototype

Reply:
Dang, missed a button!

Reply:
added new modal button.